src/lein_monolith/task/each.clj

```diff
@@ -99,22 +102,20 @@
 (defn- select-projects
   "Returns a vector of pairs of index numbers and symbols naming the selected
   subprojects."
-  [monolith subprojects project-name opts]
+  [monolith subprojects fctx project-name opts]
```

Comment:
This name `fctx` was not intelligible until I read further down the file. How about `fingerprints` instead? Alternately, is this something you could pass along inside the `monolith` map?

Reply:
I wanted to group the fingerprint-related context into one object to minimize passing around of stuff, but also felt that stuffing more things into `monolith` would make it even less obvious how to consume the fingerprint APIs. I think just renaming it to `fingerprints` would be pretty confusing and potentially overloaded with other things in the fingerprint namespace. Would `fingerprint-ctx` be an acceptable compromise?